frontend/src/tabs/authorship.pug

```diff
@@ -3,8 +3,8 @@
   .toolbar(v-if="hasCommits(info)")
     a(v-if="hasActiveFile", v-on:click="expandAll(false)") hide all file details
     a(v-else, v-on:click="expandAll(true)") show all file details
-  .title
-    a.repoName.large-font(
+  .panel-heading
+    a.groupName.large-font(
```

Comment:
perhaps name as `group-name`?

Since we are standardising between the side panels, we can remove the `.large-font` class here and just set the font size within the css of `group-name`